src/VisualStudio/Core/Test/Completion/VisualBasicCompletionSnippetNoteTests.vb

```diff
@@ -37,48 +43,52 @@ End Class]]></document>
             End Using
         End Function
 
-        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
-        Public Async Function SnippetExpansionNoteNotAddedToDescription_ShortcutIsProperSubstringOfInsertedText() As Task
-            Using state = CreateVisualBasicSnippetExpansionNoteTestState(_markup, "Interfac")
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
❕ Missing member data

Reply:
Thank you!

---
In reply to: [208790050](https://github.com/dotnet/roslyn/pull/29176#discussion_r208790050) [](ancestors = 208790050)